pages/home/home.ejs

```diff
@@ -2,108 +2,512 @@
+                <li class="list-group-item py-1">RStudio</li>
+              </ul>
+              <div class="card-footer small font-weight-bold py-1 border-0">
+                <i class="fas fa-fw fa-hourglass-half" aria-hidden="true"></i>
+                Coming soon!
+              </div>
+            </div> <!-- card -->
+            <div class="card shadow">
+              <h3 class="h6 card-header font-weight-bold py-2">
+                And more &hellip;
+              </h3>
+              <div class="card-body py-2">
+                <p class="card-text">
+                  Check out the docs to see more elements or how to roll your own!
+                </p>
+                <a class="btn btn-outline-primary btn-sm btn-block" href="#">
+                  <i class="fas fa-fw fa-shapes" aria-hidden="true"></i>
+                  Use elements
+                </a>
+                <a class="btn btn-outline-primary btn-sm btn-block" href="#">
+                  <i class="fas fa-fw fa-laptop-code" aria-hidden="true"></i>
+                  Write elements
                 </a>
-              </td>
-            </tr>
-            <% }); %>
-          </tbody>
-        </table>
+              </div>
+            </div> <!-- card -->
+          </div> <!-- card-columns -->
+        </div> <!-- container -->
+      </div> <!-- jumbotron -->
 
-        <% } %> <!-- course_instances.length -->
-      </div>
-    </div>
+      <div class="jumbotron bg-light mb-0">
+        <div class="container">
+          <h2>Ready to get started?</h2>
+          <p class="lead">
+            Start by requesting a course and creating some content. Join the Slack workspace for ad hoc
+            assistance or some random banter!
+          </p>
+          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="#">
+            <i class="fab fa-fw fa-wpforms" aria-hidden="true"></i>
+            Request your course
+          </a>
+          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="#">
+            <i class="fas fa-fw fa-laptop-code" aria-hidden="true"></i>
+            Create your content
+          </a>
+          <a class="btn btn-outline-dark btn-lg mb-2" href="#">
+            <i class="fab fa-fw fa-slack" aria-hidden="true"></i>
+            Join our community
+          </a>
+        </div> <!-- container -->
+      </div> <!-- jumbotron -->
+
+      <div class="jumbotron bg-dark text-light text-right mb-0">
+        <div class="container">
+          <h2>Want to contribute?</h2>
+          <p class="lead">
+            PrairieLearn is open source, so anyone can add features or fix bugs!
+          </p>
+          <a class="btn btn-outline-light btn-lg mr-2 mb-2" href="#">
+            <i class="fab fa-fw fa-github" aria-hidden="true"></i>
+            GitHub
+          </a>
+          <a class="btn btn-outline-light btn-lg mb-2" href="#">
+            <i class="fab fa-fw fa-slack" aria-hidden="true"></i>
+            Slack
```

Comment:
Not sure this should be here... I have an odd feeling more students will flock here than desired.

Reply:
If this is going to go forward, PL probably should consider locking down help channels for faculty or at least changing the default channels. Plus, what is the overall message output per week? Is it above ~5k with the new school adoptions mid-summer? Not sure how sustainable a ~10k message Slack will be in the future if that's the case...

Maybe it would be better to switch at this point to MS Teams so that messages could be retained? (c.f. https://answers.uillinois.edu/illinois/page.php?id=78265) I'm not sure UIUC has a license for HipChat... Otherwise, I would have suggested that as an option to avoid MS.

Reply:
Pinging @mwest1066 since this thread is a bit buried.